packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -103,11 +104,11 @@ describe( 'ToggleGroupControl', () => {
 			'Click for Delicious Gnocchi'
 		);
 
-		fireEvent.focus( firstRadio );
+		firstRadio.focus();
```

Comment:
As discussed face-to-face, we can try using `userEvent.hover()` and if that's enough, it resembles the user behavior much better, but it also is wrapped in an `act()` so it'll be compatible to React 18.

Reply:
Implemented via https://github.com/WordPress/gutenberg/pull/45627/commits/443a1279a806eae3eb34945857d39969e2176dc6